bskyembed/src/components/embed.tsx

```diff
@@ -350,3 +363,62 @@ function GenericWithImage({
     </Link>
   )
 }
+
+function StarterPackEmbed({
+  content,
+}: {
+  content: AppBskyGraphDefs.StarterPackViewBasic
+}) {
+  if (!AppBskyGraphStarterpack.isRecord(content.record)) {
+    return null
+  }
+
+  const starterPackHref = getStarterPackHref(content)
+  const imageUri = getStarterPackOgCard(content)
+
+  return (
+    <Link
+      href={starterPackHref}
+      className="w-full rounded-lg overflow-hidden border flex flex-col items-stretch">
+      <img src={imageUri} className="aspect-[1.91/1] object-cover" />
+      <div className="py-3 px-4">
+        <div className="flex space-x-2 items-center">
+          <img src={starterPackIcon} className="w-10 h-10" />
+          <div>
+            <p className="font-semibold leading-[21px]">
+              {content.record.name}
+            </p>
+            <p className="text-sm text-textLight line-clamp-2 leading-[18px]">
+              Starter pack by{' '}
+              {content.creator.displayName || content.creator.handle}
+            </p>
+          </div>
+        </div>
+        {content.record.description && (
+          <p className="text-sm mt-1">{content.record.description}</p>
+        )}
+        {!!content.joinedAllTimeCount && content.joinedAllTimeCount > 50 && (
```

Comment:
I wonder if we might want to always show this count for embeds. Or at least drop the count to something like 10 or 15?

Reply:
since you're also here @estrattonbailey, thoughts?

Reply:
I'm down to drop it to 10 or so. Should I do it for the in-app one too presumably?

Reply:
hmm, no strong opinions but i think leaving it alone in the app is fine. embeds feel more "immediately going to be seen" so think having the count show up there more easily is nice.

too bad we don't have like "joined in the last 24 hours" though.

Reply:
imo it should be consistent, whichever way we do it. bare in mind the most popular one in this UK wave (the quiet riot one) has only 67 joins